### DISCUSSION TITLE
Source code in stacktrace for C#

### DISCUSSION THREAD

**Discussion by remalkoil** [getsentry/support#57] [Support] [answered]
Hello everyone,
I've integrated Sentry in a .net application and a PHP application.
In the PHP application, the source code of the stack trace is visible. 
In the C# application, it's visible stack trace only - without source code.

Is it possible to show the source code in the stack trace of an issue for c#?

Thank you,
best regards
Daniele

**Comment by mitsuhiko** [comment, marked as answer]
The C# SDK does not yet support source context. This is a combined limitation of the source code not being available at event creation time and our pipeline not yet being able to support .NET PDBs. This might change in the future, but today these limitations exist.

For some special cases of .NET however we likely will have source support rather soon. In particular for Unity IL2CPP builds we are trying to make this work.

**Comment by bruno-garcia** [comment]
Tracked here: https://github.com/getsentry/sentry-dotnet/issues/646
